docs/cross-platform/change-log-visual-studio-tools-for-unity-mac.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Change Log (Visual Studio Tools for Unity, Mac) | Microsoft Docs"
+ms.custom: ""
+ms.date: "04/05/2018"
+ms.reviewer: ""
+ms.suite: ""
+ms.technology: vs-unity-tools
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+ms.assetid: 33a6ac54-d997-4308-b5a0-af7387460849
+author: "dantogno"
+ms.author: "v-davian"
+manager: crdun
+ms.workload:
+  - "unity"
+---
+# Change Log (Visual Studio Tools for Unity, Mac)
+Visual Studio Tools for Unity change log.
+
+## 1.5.0.1
+ Released 2018-03-28
+ 
+### New Features
+
+-   **Integration:**
+
+    -   Added support for extra templates in the Unity Project Explorer.
+
+## 1.5.0.0
+ Released 2018-03-21
+ 
+### New Features
+
+-   **Integration:**
+
+    -   Added support for detecting and attaching to Android players connected through USB.
+
+## 1.4.0.3
+ Released 2018-03-05
+ 
+### New Features
+
+-   **Project Generation:**
+
+    -   Added support for the new project generator in Unity 2018.1.
+
+-   **Integration:**
+
+    -   Added option panel for dedicated settings.
+
+## 1.4.0.2
+ Released 2018-01-24
+ 
+### Bug fixes
+
+-   **Project Generation:**
+
+    -   Fixed Mono version detection.
+
+-   **Integration:**
+
+    -   Fixed timing issues with 2018.1 and plugin activation.
+
+    -   Fixed notifications when detecting a new player.
+
+## 1.4.0.1
+ Released 2018-01-23
+ 
+### Bug fixes
+
+-   **Integration:**
+
+    -   Fixed Expand/Collapse folders on double-click
+
+## 1.4.0.0
+ Released 2017-12-13
+ 
+### New Features
+
+-   **Project Generation:**
+
+    -   Added support for .Net Standard.
```

Comment:
This should be: .NET Standard

Reply:
Good catch thanks. I fixed the windows changelog as well